Design-System/pages/wishlistPage.html

```diff
@@ -0,0 +1,485 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Vrushabh Dhatrak" />
+
+    <link rel="stylesheet" href="./../styles/main.css" />
+    <link rel="stylesheet" href="./index.css" />
+    <script
+      src="https://memorable-ui.netlify.app/script/header.js"
+      defer
+    ></script>
+    <script
+      src="https://memorable-ui.netlify.app/script/navigation.js"
+      defer
+    ></script>
+    <script src="https://code.iconify.design/2/2.1.1/iconify.min.js"></script>
+
+    <title>Wishlist</title>
+  </head>
+  <body>
+    <div class="wishlist_container">
+      <div class="header_container">
+        <header class="sticky">
+          <img
+            class="img_logo"
+            src="./../assets/apna-5.PNG"
+            alt="header_logo"
+          />
+          <a href="./../../index.html" class="logo head-2">Apna Shop </a>
+
+          <div class="toggle" onclick="toggleMenu();"></div>
+          <ul class="menu">
+            <li>
+              <a
+                class="text-3"
+                onclick="toggleMenu();"
+                href="./../../index.html"
+                >Home</a
+              >
+            </li>
+            <li>
+              <a class="text-3" onclick="toggleMenu();" href="./filterPage.html"
+                >Shop Now</a
+              >
+            </li>
+          </ul>
+          <ul class="menu hide">
+            <li>
+              <a
+                onclick="toggleMenu();"
+                class="text-1"
+                href="./../Avatar/avatar.html"
+                >Avatar</a
+              >
+            </li>
+            <li>
+              <a
+                onclick="toggleMenu();"
+                class="text-1"
+                href="./../Alert/Alert.html"
+                >Alert</a
+              >
+            </li>
+          </ul>
+
+          <ul class="menu left_auto">
+            <!-- <li><a class="text-3" onclick="toggleMenu();">Documentation</a></li>
+                <div class="btn-round">
+                  <button class="btn-primary xl-bdr-radius pad-xs">
+                    <a href="https://www.github.com/">
+                      <span class="iconify" data-icon="akar-icons:github-fill"></span>
+                    </a>
+                  </button>
+                </div> -->
+            <!--- Input with Icon -->
```

Comment:
if this is not necessary then you should consider removing comments

Reply:
That will necessary in future mobile responsiveness